common/rya.api/src/main/java/org/apache/rya/api/resolver/RyaToRdfConversions.java

```diff
@@ -40,8 +40,8 @@
  */
 public class RyaToRdfConversions {
 
-    public static URI convertURI(RyaURI uri) {
```

Comment:
RyaType is too general of an argument here.  Per our discussion, create a private method that does what you want.

Reply:
done